inclusion_connect/accounts/forms.py

```diff
@@ -57,6 +67,10 @@ def __init__(self, *args, **kwargs):
 
         self.fields["email"].widget.attrs = EMAIL_FIELDS_WIDGET_ATTRS
 
+    def save(self, commit=True):
+        self.instance.terms_accepted_at = self.instance.date_joined
```

Comment:
Plutôt que de systématiquement mettre à jour cette colonne, on peut ne le faire qu’à la création.
```suggestion
        if not self.pk:
            self.instance.terms_accepted_at = self.instance.date_joined
```

Reply:
Pardon, mal lu, je pensais que c’était dans le modèle et non le formulaire. Tu peux ignorer ce commentaire.